src/mame/layout/ferie.lay

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="pencursor">
+		<image state="1">
+			<data><![CDATA[
+				<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1" height="1" viewBox="0 0 1 1">
+					<g fill="none" stroke="#6666ff" stroke-width="0.06" stroke-opacity="1">
+						<circle cx="0.5" cy="0.5" r="0.1" />
+						<circle cx="0.5" cy="0.5" r="0.47" />
+						<line x1="0.03" y1="0.5" x2="0.4" y2="0.5" />
+						<line x1="0.6" y1="0.5" x2="0.97" y2="0.5" />
+						<line x1="0.5" y1="0.03" x2="0.5" y2="0.4" />
+						<line x1="0.5" y1="0.6" x2="0.5" y2="0.97" />
+						<line />
+					</g>
+				</svg>
+			]]></data>
+		</image>
+	</element>
+
+	<!--
+	    FIXME: Default UI font doesn't render Hiragana/Katakana.
+
```

Comment:
The default UI font is platform-dependent.  The font situation on Linux is a bit of a mess.  On macOS, due to the way MAME draws text, automatic font substitution for missing characters doesn’t happen.  However, on Windows it should be able to display kana by default.

Text rendering needs to be properly cleaned up eventually.

Reply:
Ok, I'll remove this comment then.